editor/editor_help.cpp

```diff
@@ -610,6 +616,26 @@ void EditorHelp::_update_doc() {
 		}
 	}
 
+	// Note if deprecated.
+	if (cd.is_deprecated) {
+		class_desc->push_color(get_theme_color(SNAME("warning_color"), SNAME("Editor")));
```

Comment:
Use error_color to match the above icon change suggestion:

```suggestion
		class_desc->push_color(get_theme_color(SNAME("error_color"), SNAME("Editor")));
```